client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -61,25 +50,57 @@ function ProfileInfoEdit () {
                              <Form.Control type="text" placeholder="+62123456789" />
                         </Form.Group>
 
+                        <Form.Group controlId="formWebUrl">
+                             <Form.Label>Website URL</Form.Label>
+                             <InputGroup className="mb-3">
+                                <Form.Control 
+                                    type="text"
+                                    placeholder="Team custom URL"
+                                    aria-label="Team custom URL"
```

Comment:
Is this for their own website that they build using Researchify? If so, should we place this outside of Profile page and somewhere in a separate page dedicated for website settings?

Reply:
Field deleted, this can be add outside profile settings